src/base/static/components/molecules/form-field-types/map-drawing-toolbar.scss

```diff
@@ -0,0 +1,85 @@
+.map-drawing-toolbar {
+  background-color: #888;
+  border-radius: 5px;
+  padding: 10px;
+}
+
+.map-drawing-toolbar__grid-container {
+  display: grid;
```

Comment:
css grid :tada: 

Note that this isn't fully supported in IE, but I think that should be ok :)

I just want to make sure that we are conscientious of the browsers we support.

Reply:
Yeah I thought I'd try it out here :).

This is the only place grid is used, so if we decide we don't want to use it because of IE we can pretty easily remove it.